src/feature-lifecycle/stabilization.md

```diff
@@ -1,22 +1,43 @@
 # Stabilizing features
 
-**Status:** Stub
+* **Status:** Current
+* **Last Updated:** 2022/03/10
 
 Feature stabilization involves adding `#[stable]` attributes. They may be introduced alongside new trait impls or replace existing `#[unstable]` attributes.
 
-Stabilization goes through the Libs FCP process, which occurs on the [tracking issue](./tracking-issues.md) for the feature.
+Stabilization goes through the Libs FCP[^1] process, which occurs on the [tracking issue](./tracking-issues.md) for the feature.
+
+## When is an FCP appropriate?
+
+Once an unstable feature has been well-tested with no outstanding concerns, anyone may push for it's stabilization.
```

Comment:
Rather than 'well-tested' it might make sense to focus more on the API design rather than implementation, like saying something about how the design space (alternative APIs) must have been fully explored.